src/components/views/articles/article1.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
```

Comment:
What is the difference between this file and `article2.js` ?

Reply:
Give more context.

Reply:
There are two different article components on the homepage. One type is in the landing section (article1.js) and the others are in the recent stories section and the top stories section (article2.js). The same article2.js is being used in the discover.